spring-messaging/src/main/java/org/springframework/messaging/rsocket/annotation/support/RSocketMessageHandler.java

```diff
@@ -306,7 +306,7 @@ private MessagingRSocket createResponder(ConnectionSetupPayload setupPayload, RS
 
 		s = setupPayload.metadataMimeType();
 		MimeType metaMimeType = StringUtils.hasText(s) ? MimeTypeUtils.parseMimeType(s) : this.defaultMetadataMimeType;
-		Assert.notNull(dataMimeType, "No `metadataMimeType` in ConnectionSetupPayload and no default value");
+		Assert.notNull(metadataMimeType, "No `metadataMimeType` in ConnectionSetupPayload and no default value");
```

Comment:
```suggestion
		Assert.notNull(metaMimeType, "No `metadataMimeType` in ConnectionSetupPayload and no default value");
```

Reply:
`metadataMimeType` should actually be `metaMimeType`.

I don't have "permission to push to this repository", so I'll simply perform the work locally.